images/rclone/Dockerfile

```diff
@@ -1,11 +1,16 @@
 FROM golang:1.10-alpine3.7 AS build
 
-COPY rclone /go/src/github.com/ncw/rclone/
+WORKDIR "/go/src/github.com/ncw/"
+RUN wget https://github.com/ncw/rclone/releases/download/v1.45/rclone-v1.45.tar.gz && \
```

Comment:
Might be good to have the version as an `ENV` this way it's easier to change the version when needed.

Reply:
Sure.